backend/src/apiserver/template/tekton_template.go

```diff
@@ -354,8 +356,8 @@ func (t *Tekton) injectArchivalStep(workflow util.Workflow, artifactItemsJSON ma
 					t.getObjectFieldSelector("PIPELINERUN", "metadata.labels['tekton.dev/pipelineRun']"),
 					t.getObjectFieldSelector("PODNAME", "metadata.name"),
 					t.getObjectFieldSelector("NAMESPACE", "metadata.namespace"),
-					t.getSecretKeySelector("AWS_ACCESS_KEY_ID", "mlpipeline-minio-artifact", "accesskey"),
```

Comment:
maybe instead of hard code the `mlpipeline-minio-artifact` secret name, we can parameterize it so users can point to other secret name for s3 credentials.